x-pack/legacy/plugins/canvas/i18n/angular.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const AngularStrings = {
+  CanvasRootController: {
+    getReadOnlyBadgeText: () =>
+      i18n.translate('xpack.canvas.badge.readOnly.text', {
+        defaultMessage: 'Read only',
+      }),
+    getReadOnlyBadgeTooltip: () =>
+      i18n.translate('xpack.canvas.badge.readOnly.tooltip', {
+        defaultMessage: 'Unable to save Canvas workpads',
```

Comment:
Should proper nouns like `Canvas` be a value passed in so it does not get translated?

Reply:
Agreed... will resolve.